exonum-java-binding/time-oracle/src/main/java/com/exonum/binding/time/TimeSchemaProxy.java

```diff
@@ -51,11 +52,12 @@
   }
 
   private void checkIfEnabled() {
-    MapIndex<String, InstanceSpec> serviceInstances = new DispatcherSchema(view).serviceInstances();
+    MapIndex<String, InstanceState> serviceInstances =
+        new DispatcherSchema(view).serviceInstances();
     checkArgument(serviceInstances.containsKey(name), "No time service instance "
         + "with the given name (%s) started.", name);
 
-    InstanceSpec serviceSpec = serviceInstances.get(name);
+    InstanceSpec serviceSpec = serviceInstances.get(name).getSpec();
```

Comment:
Shall we also test the 'status' to be 'active'? Though the documentation does not say what is 'pending' (@aleksuss): https://github.com/exonum/exonum-proto-sources/compare/abef50d362748589146bc06a238a736225a4d283...e236f1552f857395e556f561ccbee8ab3db2f17d#diff-fe18de47e7c24050ae38d6bbef3daa6aR99 

May be done separately — issue.